src/Bot.hs

```diff
@@ -630,18 +636,18 @@ mention =
          else Nothing <$ msg) $
   ignoreNothing markov
 
-bot :: Bot
-bot Started = do
+bot :: ChannelName -> Bot
+bot channel Started = do
```

Comment:
If we don't use this channel, why do we still pass it around?

Reply:
It's used for builtinCommands which has some channel references as well, for example on line 517 and 412.